service-front/content/guidance/People_Certificate providers.md

```diff
@@ -1,5 +1,10 @@
 # Certificate providers
 
+<div class="moj-banner moj-banner__message">
+    <p class="bold-small"><a href="https://www.gov.uk/guidance/making-and-registering-an-lpa-during-the-coronavirus-outbreak#the-certificate-provider-and-donor-conversation">Read our guidance</a>  on being a certificate provider during the corona virus outbreak</p>
```

Comment:
I know this is a tiny thing, but you have "corona virus" in each of these whereas the story says "coronavirus"

Reply:
Corrected to "coronavirus". That's a good catch Josh, thanks.

Reply:
Corrected to say "coronavirus". Thanks Josh